zuul.d/jobs.yaml

```diff
@@ -619,7 +619,7 @@
     description: |
       Run Kubernetes E2E Conformance tests against release-1.13 branch of Kubernetes
     vars:
-      go_version: '1.11.5'
+      go_version: '1.12'
```

Comment:
can 1.12 be recognized? I assume it should be `1.12.1`

Reply:
1.12 signifies 1.12.0

Reply:
1.12 worked.